person.rb

```diff
@@ -20,6 +21,11 @@ def can_use_service
     of_age || @parent_permission
   end
 
+  def rent_book(rental)
+    @rentals << rental unless @rentals.include?(rental)
+    rental.person = self
+  end
```

Comment:
Just as I have described in previous comments, since we will be accessing the rental class from here, Kindly add getters and setters for rentals😉.

Reply:
Thank you very much for the review